db/dohsql.c

```diff
@@ -1281,6 +1330,8 @@ int dohsql_end_distribute(struct sqlclntstate *clnt, struct reqlogger *logger)
     clnt->conns = NULL;
     free(conns);
 
+    _rem_parallel_load();
```

Comment:
There's a couple of early returns that don't _rem_parallel_load

Reply:
Only one return, if failed to run recover_deadlock.  That is probably unrecoverable though. We'll add the call to _rem_parallel_load anyway